website/pages/component/confirmation-modal/guidelines.mdx

```diff
@@ -1,12 +1,10 @@
-import { css } from '@emotion/css';
-import { Body, Link } from '@leafygreen-ui/typography';
+import { css } from "@emotion/css";
```

Comment:
We're using single quotes everywhere else, we should probably stick to those. I've noticed there are a few files where they've been switch to double quotes.